src/content/lodash/util/noop/notes.md

```diff
@@ -0,0 +1 @@
+[mdn](https://developer.mozilla.org/en-US/docs/Glossary/Function)
```

Comment:
I'm not sure that this function adds any value actually

Reply:
I agree with you, please remove it :)